src/main/java/net/javadiscord/javabot/systems/help/HelpListener.java

```diff
@@ -54,40 +48,58 @@ public class HelpListener extends ListenerAdapter implements ButtonHandler {
 		newThreadChannels = new HashSet<>();
 	}
 
+	private final UserPreferenceService preferenceService;
 	private final BotConfig botConfig;
 	private final HelpAccountRepository helpAccountRepository;
 	private final HelpTransactionRepository helpTransactionRepository;
 	private final HelpExperienceService experienceService;
 	private final DbActions dbActions;
-	private final String[][] closeSuggestionDetectors = {
-			{"close", "post"},
-			{"close", "thread"},
-			{"close", "question"},
-			{"problem","solv"},
-			{"issue","solv"},
-			{"thank"}
-	};
+	private final String[][] closeSuggestionDetectors = {{"close", "post"}, {"close", "thread"}, {"close", "question"}, {"problem", "solv"}, {"issue", "solv"}, {"thank"}};
 	private final long SUGGEST_CLOSE_TIMEOUT = 5 * 60_000L;//5 minutes
 	private final Map<Long, Long> recentlyCloseSuggestedPosts = new LinkedHashMap<>(8, 0.75f, true) {
 		@Override
 		protected boolean removeEldestEntry(Map.Entry<Long, Long> eldest) {
 			return System.currentTimeMillis() > eldest.getValue() || size() >= 32;
 		}
 	};
-	private final UserPreferenceService preferenceService;
+	private final AutoCodeFormatter autoCodeFormatter;
+
+	/**
+	 * The constructor of this class, which is set corresponding to lombok.
+	 *
+	 * @param autoMod                   automod reference.
+	 * @param preferenceService         UserPreferenceService reference
+	 * @param botConfig                 BotConfig reference
+	 * @param helpTransactionRepository HelpAccountRepository reference
+	 * @param helpAccountRepository     HelpAccountRepository reference
+	 * @param dbActions                 DbActions reference
+	 * @param experienceService         ExperienceService reference
+	 */
+	public HelpListener(AutoMod autoMod, UserPreferenceService preferenceService, BotConfig botConfig, HelpAccountRepository helpAccountRepository, HelpTransactionRepository helpTransactionRepository, HelpExperienceService experienceService, DbActions dbActions) {
```

Comment:
If `AutoCodeFormatter` is a Spring bean, this constructor can be replaced by lombok.